### PR TITLE
Change theme color

### PR DESCRIPTION
**Pull request by pklion**
#21

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -8,6 +8,11 @@
   <meta name="description" content="Description">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
   <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/lib/themes/vue.css">
+  <style>
+    html {
+      --theme-color:#c4a604;
```
Comment (chingucoding) [inline docs/index.html:13]: This has a contrast ratio of 2.38, normal text (which this resource would be used for) should have a ratio of 4.5. @yaichenbaum Is this the "theme" color to use and if so what do you suggest to do with this?

Comment (yaichenbaum) [reply docs/index.html:13]: @chingucoding I was thinking we should use something closer to `#E89609`.

Comment (chingucoding) [reply docs/index.html:13]: Hmm I see. The contrast ratio isn't ideal too @yaichenbaum:

Screenshot showing bright orange text on white background

Not sure if it's a sacrifice worth making but given that links will use this color, having good contrast ratios should be a goal.

Comment (yaichenbaum) [reply docs/index.html:13]: @chingucoding That was just a general idea for which direction to move in, do you have any suggestions for a color with better contrast ratios?

Comment (chingucoding) [reply docs/index.html:13]: I see. For 3:1 ratio you could use `#b37407`, for 4.5:1 you could use `#a66907` or `#a36507`. Unfortunately, dark orange is just brown...

Comment (yaichenbaum) [reply docs/index.html:13]: Let's try `#a66907`, and if we want, we can change it later.

Comment (chingucoding) [reply docs/index.html:13]:
```suggestion
      --theme-color:#a66907;
```

To make it easier for you @pklion

Comment (pklion) [reply docs/index.html]: Oh, I didn't think about the contrast ratio.
Thank you!

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**